addons/visibility/config.cpp

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"zen_common"};
+        author = ECSTRING(main,Author);
+        authors[] = {"Brett"};
```

Comment:
#47 had `SynxieBrett`

Reply:
I know, I'm going to change #47 to match the AUTHORS.txt